### PR TITLE
docs: Add note explaining installation issues

### PR DESCRIPTION
**Pull request by CalMacCQ**
Adding a note to the docs explaining the installation issues in #1513 

Screenshot 2024-08-02 at 13 25 48

### CODE REVIEW

File: pytket/docs/index.rst [hunk 1]
```diff
@@ -13,6 +13,10 @@ Windows. To install, run
     pip install pytket
 
 
+.. admonition:: Known issue installing pytket
+   :class: attention
+
+   Due to the removal of the `types-pkg_resources` package from pypi there will likely be issues when installing old versions of pytket. It is recommend to use pytket `>1.31` where possible.
```
Comment (cqc-melf) [inline pytket/docs/index.rst:19]:
```suggestion
   Due to the removal of the `types-pkg_resources` package from pypi there will likely be issues when installing old versions of pytket. It is recommend to use pytket `>=1.31` where possible.

**Comment by cqc-melf** [comment]
And it might be good to add something about how to solve that for the older versions?

**Comment by cqc-melf** [comment]
@CalMacCQ are you still working on this?

File: pytket/docs/index.rst (hunk 1 above)
Comment (CalMacCQ) [reply pytket/docs/index.rst]: fixed

File: pytket/docs/index.rst [hunk 2]
```diff
@@ -13,6 +13,16 @@ Windows. To install, run
...
 
+.. admonition:: Known issue installing pytket (Added 2nd August 2024)
+   :class: attention
+
+   Due to the removal of the `types-pkg_resources` package from pypi there will likely be issues when installing old versions of pytket. It is recommend to use pytket `>=1.31` where possible.
+
+   If you require using a version of pytket older than 1.31 then you can try the following.
+
+   
+    `pip install types-pkg-resources==0.1.3 pytket==1.29.1`
```
Comment (cqc-melf) [inline pytket/docs/index.rst:24]:
```suggestion
    `pip install types-pkg-resources==0.1.3 pytket==<set version>`

Comment (cqc-melf) [reply pytket/docs/index.rst:24]: Maybe like this? Point out that this should work with older versions, too?

**Review by cqc-melf — APPROVED** [review]

**Comment by cqc-melf** [comment]
@CalMacCQ can I merge this now?

**Pull request merged**